### ISSUE TITLE
shared_buffers and effective_cache_size config error

### ISSUE THREAD

**Issue by twk3** [openshift/postgresql#109]
Error output: 
```
waiting for server to start....LOG:  invalid value for parameter "shared_buffers": "128M"
HINT:  Valid units for this parameter are "kB", "MB", "GB", and "TB".
LOG:  invalid value for parameter "effective_cache_size": "256M"
HINT:  Valid units for this parameter are "kB", "MB", "GB", and "TB".
FATAL:  configuration file "/var/lib/pgsql/openshift-custom-postgresql.conf" contains errors
.... stopped waiting
pg_ctl: could not start server
Examine the log output.
```

Looks like computed settings based on memory aren't quite right: https://github.com/openshift/postgresql/blob/master/9.4/root/usr/share/container-scripts/postgresql/common.sh#L13-L15

**Comment by praiskup** [comment]
Should be fixed by #110.

**Comment by mnagy** [comment]
Yeah, closing.
